pitaya-sharp/NPitaya/NPitaya.csproj

```diff
@@ -13,12 +13,13 @@
         <RepositoryUrl>https://github.com/topfreegames/libpitaya-cluster</RepositoryUrl>
         <PackageTags>pitaya,server,games,framework,rpc</PackageTags>
         <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-        <TargetFrameworks>net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Was this necessary in order to use prometheus?

Reply:
I reintroduced `netstandard2.0`. However, I cannot compile stuff when adding `net461`. Is it strictly necessary? If so, how are you able to use it in your environment (macOS, I suppose)?

Reply:
I think you can reintroduce net461 and test with the ci if it is compiling correctly.